include/psa/crypto_values.h

```diff
@@ -1817,13 +1817,37 @@
  * This key derivation algorithm uses the following inputs, which must be
  * passed in the order given here:
  * - #PSA_KEY_DERIVATION_INPUT_SEED is the seed.
+ * - #PSA_KEY_DERIVATION_INPUT_OTHER_SECRET is the other secret for the
```

Comment:
Please update (remove?) the section “Custom key derivations for mixed-PSK handshake” `docs/architecture/psa-migration/psa-limitations.md`, since this is now implemented.